src/renderer/components/ft-icon-button/ft-icon-button.js

```diff
@@ -60,25 +60,48 @@ export default Vue.extend({
   },
   methods: {
     toggleDropdown: function () {
-      $(`#${this.id}`)[0].style.display = 'inline'
-      $(`#${this.id}`).focus()
+      const $el = $(`#${this.id}`)
```

Comment:
Prefixing JQuery objects with a "$" is commonly done, but I'm not sure if this variable naming style is used at all currently in FreeTube. If not, I would advise sticking with the style used in the rest of the repo.